pallets/nomination/src/lib.rs

```diff
@@ -245,8 +246,8 @@ impl<T: Config> Pallet<T> {
 		}
 
 		ext::vault_registry::pool_manager::withdraw_collateral::<T>(
-			&vault_id,
```

Comment:
`vault_id and `nominator_id` are both reference variables, `&` is not needed anymore. 

(From Clippy)